lib/models.js

```diff
@@ -1024,6 +1048,10 @@ class IncludesTypeConstraint extends Constraint {
       "onValue": this.onValue
     }, super.toJSON())
   }
+
+  toString() {
+    return `IncludesTypeConstraint (${this.card.toString()} ${this.isA.fqn})`
```

Comment:
If there is a `path`, please represent it in `toString`.  Similarly, if `onValue` is set, please indicate that too.